pyflp/plugin.py

```diff
@@ -67,6 +68,23 @@ class _FruityBalanceStruct(StructBase):
     PROPS = {"pan": "I", "volume": "I"}
 
 
+class _FruityBloodOverdriveStruct(StructBase):
+    PROPS = dict.fromkeys(
+        (
+            "plugin_marker",
+            "pre_band",
+            "color",
+            "pre_amp",
+            "x100",
+            "post_filter",
+            "post_gain",
+            "unknown_event_first",
```

Comment:
Rename to `_u1` and `_u2` respectively

Reply:
Renamed in 6b9e1674228ae979bd83fdf22a38a81ed215cc55